crates/tinyvec-embed/src/db.rs

```diff
@@ -242,6 +252,13 @@ impl Value {
             _ => Err(Error::ValueNotString(self.to_owned())),
         }
     }
+
+    pub fn inner_value(&self) -> Result<usize> {
+        match self {
+            Self::Usize(s) => Ok(s.to_owned()),
+            _ => Err(Error::ValueNotString(self.to_owned())),
+        }
+    }
```

Comment:
Sorry I think I wasn't clear when I meant rollback to the previous change, I meant you can go back to `impl TryInto for &Value`